pkg/spec/parse.go

```diff
@@ -183,9 +183,9 @@ func parseDevice(device string) (string, string, string, error) { //nolint
 	return src, dst, permissions, nil
 }
 
-// validDeviceMode checks if the mode for device is valid or not.
+// ValidDeviceMode checks if the mode for device is valid or not.
```

Comment:
Is ValidDeviceMode actually used within cri-o PR other then called from ParseDevice?  If not, then leave this lower case.

Reply:
Yes. `ValidDeviceMode ()` also has been used out of ParseDevice

Reply:
I am late to the game but as we're about to change the identifier of this function: can we rename it to `IsValidDeviceMode() bool`? It's somehow convention to have the `Is` prefix to indicate a boolean return value.

Reply:
@vrothberg make sense, :ok_hand:

Reply:
Yes please, missed that I did.  +1 to @vrothberg